src/main/java/de/retest/recheck/review/ignore/matcher/ElementClassMatcher.java

```diff
@@ -0,0 +1,52 @@
+package de.retest.recheck.review.ignore.matcher;
+
+import java.util.Objects;
+import java.util.regex.MatchResult;
+import java.util.regex.Pattern;
+
+import de.retest.recheck.review.ignore.io.RegexLoader;
+import de.retest.recheck.ui.descriptors.Element;
+
+public class ElementClassMatcher implements Matcher<Element> {
+
+	public static final String CLASS_KEY = "class";
+
+	private final String classValue;
+
+	public ElementClassMatcher( final Element element ) {
+		this( Objects.toString( element.getIdentifyingAttributes().get( CLASS_KEY ), "" ) );
+	}
+
+	private ElementClassMatcher( final String classValue ) {
+		this.classValue = classValue;
+	}
+
+	@Override
+	public boolean test( final Element element ) {
+		final String classValue = element.getIdentifyingAttributes().get( CLASS_KEY );
+		return classValue != null && classValue.matches( this.classValue );
```

Comment:
Not relevant as it only appears once (see above)?

Reply:
I am aware that our Attributes essentially represent a `Map<String, Object`>. I built upon the assumption of one big string, where it essentially represents a sequence of classes, concatenated by a space (as described above).

Thus, if we want to find a single class within the sequence, we need to process it as such a `Set<String>`.

Take Selniums `By.className` for example:
> Find elements based on the value of the "class" attribute. If an element has multiple classes, then this will match against each of them. For example, if the value is "one two onone", then the class names "one" and "two" will match.

I think that we need to match this behavior.

Reply:
I thought `matches` handles this, I now use `contains`. What do you think?